.scalafmt.conf

```diff
@@ -0,0 +1,4 @@
+version = 3.6.1
+
+runner.dialect = scala213
+maxColumn = 80
```

Comment:
80 chars wide, on 2023 screens is very small...

Reply:
Agreed it is small! But there are some good reasons for it:

* it is how we format our TS/JS code (https://github.com/guardian/csnx/blob/main/libs/%40guardian/prettier/index.js#L6) and consistent with Prettier [recommendations](https://prettier.io/docs/en/options.html#print-width)
* human eyes find this width easy to scan (think of paperback books which are typically sized for easy reading)
* easier to have two editors/windows side-by-side
* easier to see entire lines without wrapping in Github code reviews
* (finally, and less important, we've already applied this to all of our DevX scala repos now and would need to reformat them again to change it)